.travis.yml

```
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - '8'
+  - '12'
```

I think you can let multiple target if there are working:
```suggestion
  - '8'
  - '10'
  - '12'
```